python/mlc_llm/serve/entrypoints/openai_entrypoints.py

```diff
@@ -39,11 +39,12 @@
 
 
 @app.get("/v1/models")
-async def request_models():
+async def request_models(request: fastapi.Request):
     """OpenAI-compatible served model query API.
     API reference: https://platform.openai.com/docs/api-reference/models
     """
-    return ListResponse(data=[ModelResponse(id=model) for model in ServerContext.get_model_list()])
+    server_context: ServerContext = request.scope["server_context"]
+    return ListResponse(data=[ModelResponse(id=model) for model in server_context.get_model_list()])
```

Comment:
Use `ServerContext.current()` for these places. And we do not need to introduce the `request` parameter.